internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/DaVinciClientTest.java

```diff
@@ -1004,71 +1001,58 @@ public void testReadCompressedData(CompressionStrategy compressionStrategy) thro
    * For the local P2P testing, need to setup two different directories and ports for the two Da Vinci clients in order
    * to avoid conflicts.
    */
-  @Test(timeOut = 2 * TEST_TIMEOUT, enabled = false)
```

Comment:
since i have implemented what you suggested below should i still have it as enabled false or just keep it on now since theres an assert

Reply:
I think either way is fine. The validation added for now does not truly validated the blob transfer happens, it only validates snapshot is taken. We will have to fix that critical OffsetRecord transfer logic and such before we can truly assert on things. Even at that time, for integration test IDK what is really a great way to assert as even if blob transfer failed Kafka ingestion will kick in so data will be there. Maybe we can assert that Kafka ingestion metric remains 0. Or @adamxchen  do you think if there is any other way to validate this behavior?

But for now, this gap does not block this PR to be merged. I plan to further improve the class to make it more general for different DVC use case.

Reply:
since either way is fine then once all the github checks are passed ill set it to false so that it doesnt add extra time to the CI

Reply:
Sure. If it doesn't add much time, I'm okay to leave it this way. Can we leave a comment in the description (line 1001-1003) though? I don't wanna give people a wrong impression that this integ test is working.

> for integration test IDK what is really a great way to assert as even if blob transfer failed Kafka ingestion will kick in so data will be there.

This is a good question. I think we can verify the`IDENTITY` file in the rocksdb partition. Per [the doc](https://github.com/facebook/rocksdb/wiki/MANIFEST), the IDENTITY should contain an unique DB id - we need to verify whether it would be changed during the lifecycle -  and if it remain the same, we can use it to verify the snapshots are actually copied from one DVC client to another.

Reply:
Should I just leave a TODO?
Like TODO: Add asserts to the test to validate if the blob transfer was performed

Reply:
Yes TODO would be good

Reply:
done in latest commit